app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
Why not, I really don't care. @skateman ?

Anyway, that can be done anytime even in a separate PR.